db/seeds.rb

```diff
@@ -284,3 +284,16 @@
     lrt
   ).build_requirement_template_from_existing(overrides)
 end
```

Comment:
note to merge in the PR that adds this to develop, and make sure this code isn't duplicated when merging/rebasing